frontend/app/templates/work_packages/attachments-edit.html

```diff
@@ -12,7 +12,7 @@ <h3 class="attributes-group--header-text">
        data-ngf-select
        class="work-package--attachments--drop-box"
        data-ngf-multiple="true"
-       data-ngf-change="instantUpload()"
+       data-ngf-change="filesChanged($files, $file, $event, true)"
```

Comment:
You are using a flag argument here and as Uncle Bob tells us:
> Boolean arguments loudly declare that the function does more than one thing. They are confusing and should be eliminated.

I could especially understand the problem in this case, because I read the usage before reading the definition of the method. I had no idea what that dangling `true` meant.

Also `filesChanged` is not a very descriptive name, I guess it is some name convention for an event handler? Such as "do this when files changed"...

I'd rather call it something like `filterFiles` and `uploadFilteredFiles`...